src/index.ts

```diff
@@ -389,7 +394,8 @@ function findHotkey(toolbar: Element, key: string): HTMLElement | null {
 
 function shortcut(toolbar: Element, event: KeyboardEvent) {
   if ((event.metaKey && modifierKey === 'Meta') || (event.ctrlKey && modifierKey === 'Control')) {
-    const button = findHotkey(toolbar, event.key)
+    const key = event.shiftKey ? event.key.toUpperCase() : event.key
+    const button = findHotkey(toolbar, key)
```

Comment:
I don't think this change is needed. I can revert this block locally and the tests still run green.

The case-sensitivity is already backed in since we do strict equality of the key pressed and the `hotkey` attribute.

Reply:
@koddsson I must be doing something wrong because this test fails for me on `main`.

```
code
        ✔ surrounds a line with backticks if you click the code icon
        ✖ surrounds a line with backticks via hotkey
        ✔ surrounds multiple lines with triple backticks if you click the code icon
        ✔ removes backticks from a line if you click the code icon again
        ✔ removes triple backticks on multiple lines if you click the code icon
      links
        ✔ inserts link syntax with cursor in description
        ✔ selected url is wrapped in link syntax with cursor in description
        ✔ cursor on url is wrapped in link syntax with cursor in description
        ✔ selected plan text is wrapped in link syntax with cursor in url
      images
        ✔ inserts image syntax with cursor in description
        ✔ selected url is wrapped in image syntax with cursor in description
        ✔ cursor on url is wrapped in image syntax with cursor in description
        ✔ selected plan text is wrapped in image syntax with cursor in url
      header
        ✔ inserts header syntax with cursor in description
        ✔ inserts header 1 syntax with cursor in description
        ✔ does not insert header for invalid level

Finished in 0.013 secs / 0.091 secs @ 22:03:40 GMT-0400 (Eastern Daylight Time)

SUMMARY:
✔ 86 tests completed
✖ 1 test failed

FAILED TESTS:
  markdown-toolbar-element
    after tree insertion
      code
        ✖ surrounds a line with backticks via hotkey
          Chrome Headless 89.0.4389.90 (Mac OS 11.1.0)
        AssertionError: expected '`|puts \'Hello, world!\'|`' to equal '|puts \'Hello, world!\'|'
            at Context.<anonymous> (test.js:621:16)
```

Reply:
What does it do for you on the [example page](https://github.github.com/markdown-toolbar-element/examples/)?

If I `cmd-b` or `cmd-B`, it bolds the text.  This is true for me on Chrome, Safari, and Firefox.

Reply:
@koddsson Wait.  Scratch my first comment above.  I was testing the wrong thing.  Let me look into that in the morning, but my second comment ☝️ still stands.  I'm curious how this behaves for you.

Reply:
> What does it do for you on the [example page](https://github.github.com/markdown-toolbar-element/examples/)?
> 
> If I `cmd-b` or `cmd-B` (cmd-shift-b), it bolds the text. I would expect it only to respond to a lowercase `b`.

<kbd>⌘</kbd>+<kbd>B</kbd> is a global shortcut on my mac that launches a application 😄 

But yeah if I turn that off then I can bold with both. This is unexpected but apparently, if you are holding the meta key down `KeyboardEvent.key` will always be lowercase.

I made a little site to test out the `keydown` handler [here](https://js-is.glitch.me/test/uppercase-key-when-holding-metakey.html).

Getting the tests to fail when this block is removed would be good though 👍🏻

Reply:
Oh yeah... this is a specific edge case with macos reporting lower case keys for specifically `cmd+shift`. See https://bugs.webkit.org/show_bug.cgi?id=174782 & https://bugs.chromium.org/p/chromium/issues/detail?id=747358

Reply:
Good finds and information, @koddsson and @keithamus.  I'll work on getting a breaking test on this.

Reply:
- [X] add [breaking test](https://github.com/github/markdown-toolbar-element/pull/37/commits/5c1807c8194563d91044b57e3ae5acb63c586dec) for lower/uppercase hotkey